File: SofiForce.Models/Models/EntityModels/PromotionOutcomeModel.cs
using System;

namespace Models
{
    public class PromotionOutcomeModel
    {
        public int OutcomeId { get; set; }
        public int? PromotionId { get; set; }
        public bool? IsActive { get; set; }
        public int? Count { get; set; }
        public int? DisplayOrder { get; set; }
        public string ItemCode { get; set; }
        public string ItemName { get; set; }

        public decimal? Slice { get; set; }
        public decimal? Value { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/PromotionOutcomeModel.ts
export class PromotionOutcomeModel {
    outcomeId: number;
    promotionId: number | null;
    isActive: boolean | null;
    count: number | null;
    displayOrder: number | null;
    itemCode: string;
    slice: number | null;
    value: number | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;
}